docs/ManualDeploymentGuideScalarDBServerOnEKS.md

```diff
@@ -0,0 +1,200 @@
+
+### Steps
+
+* Follow [Set up a database guide](./SetupDatabase.md) to set up a database for Scalar DB.
+
+## Step 3. Configure EKS
+
+This section shows how to create an EKS cluster and 2 managed node groups(one for Scalar DB server and envoy and one for logs and metrics collection) for the Scalar DB server and monitor agent deployment.
+
+### Prerequisites
+
+Install the following tools on your bastion for controlling the EKS cluster:
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html): In this guide, AWS CLI is used to create a kubeconfig file to access the EKS cluster.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/): Kubernetes command-line tool to manage EKS cluster. Kubectl 1.19 or higher is required.
+
+### Requirements
+
+* You must create a Kubernetes cluster with version 1.19 or higher for Scalar DB server deployment.
+* You must create a node group with the label, key as `agentpool` and value as `scalardbpool` for Scalar DB server deployment.
+* You must add a rule in the EKS security group to **enable HTTPS access (Port 443)** to the private EKS cluster from the bastion server.
+
+### Recommendations
+
+* You should create 3 nodes in each node group for high availability in the production.
+* You should configure Cluster autoscaler based on the [AWS official guide](https://docs.aws.amazon.com/eks/latest/userguide/cluster-autoscaler.html) to scale the number of nodes in your cluster.
+
+### Steps
+
+* Create an Amazon EKS cluster based on the [AWS official guide](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html).
+* Create a managed node group based on the [AWS official guide](https://docs.aws.amazon.com/eks/latest/userguide/create-managed-node-group.html) with the above requirements and recommendations.
+* Configure kubectl to connect to your Kubernetes cluster using the `aws eks update-kubeconfig` command.
+
+   ```console
+   aws eks --region <region-code> update-kubeconfig --name <cluster_name>
+   ```
+
+## Step 4. Install Scalar DB server
+
+After creating a Kubernetes cluster, the next step is to deploy the Scalar DB server into the EKS cluster.
+This section shows how to install the Scalar DB server to the EKS cluster with [Helm charts](https://github.com/scalar-labs/helm-charts).
+
+### Prerequisites
+
+Install the Helm on your bastion server to deploy helm charts:
+
+* [Helm](https://helm.sh/docs/intro/install/): Helm is a command-line tool to manage releases in the EKS cluster. In this tutorial, it is used to deploy the Scalar DB server helm chart to the EKS cluster. Helm version 3.5 or the latest is required.
+
+### Steps
+
+The following steps show how to install the Scalar DB server on EKS:
+
+1. Download the following Scalar DB server configuration files from the [scalar-kubernetes repository](../conf).
+   Note that they may be versioned in the future, so you might want to change the branch to use a proper version.
+    * scalardb-custom-values.yaml
```

Comment:
> 1. About tolerations
> 2. About podAntiAffinity

For both `tolerations` and `podAntiAffinity`. There was no problem without adding these configurations.

> Server manifest (reasons that there is different between scalardb-custom-values.yaml and scalardl-custom-values.yaml), could you please tell me that?

There was no specific reason for not adding these.
I had missed adding these configurations and hence not tested with adding them.
But it may cause an issue in the future if deployed other services in nodes.
Thank you for noting this.
I will update the configuration file `scalardb-custom-values.yaml` and I will create as a new PR for that after testing it.